pkg/errors/errors.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018-2020 CERN
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// In applying this license, CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+package errors
+
+import (
+	"fmt"
+	"path"
+	"runtime"
+	"strings"
+
+	"github.com/pkg/errors"
```

Comment:
Can we rely instead on the std lib errors package?

import "errors" ?

Reply:
The stdlib package doesn't have the wrap functionality.

Reply:
Should be possible with fmt package together, check the documentation for the examples:

```
A simple way to create wrapped errors is to call fmt.Errorf and apply the %w verb to the error argument:

errors.Unwrap(fmt.Errorf("... %w ...", ..., err, ...))
```